middleware/app/Http/Controllers/Api/V1/Controller.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Http\Api\V1\Controllers;
```

Comment:
I think according to PSR-0 this has to be `namespace App\Http\Controllers` to match the directory structure. Obv this would change the `use App\Http\Api\V1\Controller` statements in the controllers in the `Api\V1` directory tho